test_common/harness/imageHelpers.cpp

```diff
@@ -3810,18 +3809,50 @@ bool check_minimum_supported(cl_image_format *formatList,
 			}
 		}
 	}
+}
 
-    for (auto &format: formatsToSupport)
-    {
-        if( !find_format( formatList, numFormats, &format ) )
-        {
-            log_error( "ERROR: Format required by OpenCL %s is not supported: ", version.to_string().c_str() );
-            print_header( &format, true );
-            passed = false;
-        }
-    }
+bool is_image_format_required(cl_image_format format,
+                              cl_mem_flags flags,
+                              cl_mem_object_type image_type,
+                              cl_device_id device)
+{
+	std::vector<cl_image_format> formatsToSupport;
+	build_required_image_formats(flags, image_type, device, formatsToSupport);
+
+	for (auto &formatItr: formatsToSupport)
```

Comment:
You could use `std::find` though it's not that much of an improvement.

Reply:
I originally had that, but didn't want to also overload `==` for `cl_image_format`.

Reply:
I wasn't expecting this to be required. If it is, it's probably not worth it then.